openmdao/visualization/connection_viewer/viewconns.py

```diff
@@ -197,8 +197,10 @@ def view_connections(root, outfile='connections.html', show_browser=True,
         s = s.replace("<tabulator_style>", tabulator_style)
         f.write(s)
 
-    if notebook:
+    if notebook and not colab:
         display(IFrame(src=outfile, width=1000, height=1000))
+    else:
```

Comment:
With this change, if notebook is False,  then we execute the else branch and call `display(HTML(outfile))`, whereas we didn't do anything before. Is this expected?

Reply:
I found this in `n2_viewer` as well.. on my branch, I have changed the logic as follows:
```
    if notebook:
        # display in Jupyter Notebook
        if not colab:
            display(IFrame(src=outfile, width=1000, height=1000))
        else:
            display(HTML(outfile))
    elif show_browser:
        # open it up in the browser
        from openmdao.utils.webview import webview
        webview(outfile)
```
I think this is what we want?

Reply:
Yeah that's a cleaner edit. Added